### PR TITLE
Fix/NSA-147/layout for small screen resolution

### PR DESCRIPTION
**Pull request by andrey-ponamarev**
**Related to task:**  https://oat-sa.atlassian.net/browse/NSA-147
**Description:**  Change layout to use flexbox model for making content available on the itesm authoring page for a small screen resolutions.
Screenshot 2020-10-08 at 15 20 06

**Steps to reproduce:**
1. Login as admin
2. Change the browser size to have a few rows in the header 
2. Go to Items tab
3. Create a new item and go to Authoring
4. Add any interaction to the canvas (f.e. choice) and go to response mode
5. Click "Add an outcome" button until it reaches the bottom of the sidebar

Actual result: "Add an outcome" button is not accessible

### CODE REVIEW

File: views/scss/inc/_header-footer.scss [hunk 1]
```diff
@@ -312,9 +306,8 @@ body > footer {
 }
 
 .content-wrap {
-    min-height: 100%;
-    // equal to footer height
-    margin-bottom: -1px * $footerHeight;
+    flex: 1;
+    overflow-y: scroll;
```
Comment (bziondik) [inline views/scss/inc/_header-footer.scss:310]: Is `overflow-y: scroll;` needed?

**Review by bziondik — COMMENTED** [review]
Works for me. Only one comment

**Review by bziondik — APPROVED** [review]
- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [x]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Review by adriloma — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] bumping versions are applied (if applicable)

File: views/scss/inc/_header-footer.scss (hunk 1 above)
Comment (jsconan) [reply views/scss/inc/_header-footer.scss:310]: This is adding a permanent vertical scroll bar.

**Review by jsconan — CHANGES_REQUESTED** [review]
I only made a quick check. It seems to work.
However, I'm concerned about the possible regression it will add. Especially since a permanent vertical scroll bar is added.
If the purpose is to improve the experience on small screen size, can't you consider using media query instead? This would avoid impacting everyone.

**Comment by andrey-ponamarev** [comment]
The issue is fixed https://github.com/oat-sa/extension-tao-itemqti/pull/1562

**Pull request closed without merging**